pkg/manager/manager.go

```diff
@@ -498,7 +498,7 @@ func versionForRefs(refs *prowapiv1.Refs) string {
 		return ""
 	}
 	if refs.BaseRef == "master" || refs.BaseRef == "main" {
-		return "4.12.0-0.latest"
+		return "4.13.0-0.latest"
```

Comment:
We will typically take care of this when the time comes, but if you want us to use this PR, there are a few more changes that will need to be added if it's to eventually be merged.  You can search the code for `4.12` and bump them all.  There is another spot that points to `4.11`.  It will need to be bumped to `4.12` accordingly.